templates/expo-template-default/scripts/reset-project.js

```diff
@@ -74,11 +87,26 @@ const moveDirectories = async () => {
 
     console.log("\n✅ Project reset complete. Next steps:");
     console.log(
-      "1. Run `npx expo start` to start a development server.\n2. Edit app/index.tsx to edit the main screen.\n3. Delete the /app-example directory when you're done referencing it."
+      `1. Run \`npx expo start\` to start a development server.\n2. Edit app/index.tsx to edit the main screen.${
+        userInput === "y"
+          ? `\n3. Delete the /${exampleDir} directory when you're done referencing it.`
+          : ""
+      }`
     );
   } catch (error) {
-    console.error(`Error during script execution: ${error}`);
+    console.error(`❌ Error during script execution: ${error.message}`);
   }
 };
 
-moveDirectories();
+rl.question(
```

Comment:
can we make this default to `y` so you can just hit return and it'll do the thing, without you needing to enter anything?

Reply:
Added a default `y`

[15df176](https://github.com/expo/expo/pull/34200/commits/15df176e63e8b7bf0b9c8aa2df7930f7a953a9e3)

https://github.com/user-attachments/assets/18d026ce-2169-4605-8930-fe2ccb27de15